ovh/client.py

```diff
@@ -46,17 +46,9 @@
     # Python 3
     from urllib.parse import urlencode
 
-from .vendor.requests import request, Session
-from .vendor.requests.packages import urllib3
-from .vendor.requests.exceptions import RequestException
-
-# Disable pyopenssl. It breaks SSL connection pool when SSL connection is
-# closed unexpectedly by the server. And we don't need SNI anyway.
-try:
-    from .vendor.requests.packages.urllib3.contrib import pyopenssl
-    pyopenssl.extract_from_urllib3()
-except ImportError:
-    pass
+from requests import request, Session
+from requests.packages import urllib3
+from requests.exceptions import RequestException
 
 # Disable SNI related Warning. The API does not rely on it
```

Comment:
```suggestion
```

I will have to remove that one, as the API do now rely on it.

Reply:
Il push a new commit to remove comment and SNI warning deactivation.